tests/components/sonarr/test_sensor.py

```diff
@@ -526,7 +527,7 @@ def test_diskspace_paths(self, req_mock):
             device.update()
             assert "263.10" == device.state
             assert "mdi:harddisk" == device.icon
-            assert "GB" == device.unit_of_measurement
+            assert DATA_GIGABYTES == device.unit_of_measurement
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5001170)